### DISCUSSION TITLE
What do the build log numbers mean?

### DISCUSSION THREAD

**Discussion by fostimus** [gatsbyjs/gatsby#38537] [Help] [answered]
E.g. `0.315s - 65/65 206.31/s`

- 0.315s - took 0.315s to finish
- 65/65 - ?
- 206.31/s - ?

**Comment by violy** [comment, marked as answer]
I thinks it's (according your exemple) : 

- A → 0.315s → task total duration time
- B → 65/65 → occurences numbers of the task
- C → 206.31s → single occurence average duration time

```
65 / 0.315 = 206.31s
B / A = C 
```
